### PR TITLE
Update part of the keys in the schema guide

### PR DESCRIPTION
**Pull request by sdruskat**
**Related issues**

Refs: #206

**Describe the changes made in this pull request**

- Update the schema guide to include the portion of keys assigned to me
- Fix typo *URL to* to *URL of* (_locator of_, not _locator to_)
- Fix description of `repository-code` in schema to include _dataset_

**Instructions to review the pull request**

- Read carefully and check against schema

### CODE REVIEW

File: schema-guide.md [hunk 1]
```diff
@@ -313,14 +313,15 @@ Array of [`definitions.reference`](#definitionsreference) objects.
 
 ### `repository`
 
-- type: ...
+- type: [`definitions.url`](#definitionsurl)
```
Comment (sdruskat) [inline schema-guide.md:316]: @jspaaks Is this the way we want to reference "simple" definitions (as compared to *objects*)? I think it's the most concise and clear way.

Comment (jspaaks) [reply schema-guide.md:317]: Good with me

File: schema-guide.md [hunk 2]
```diff
@@ -313,87 +314,93 @@ Array of [`definitions.reference`](#definitionsreference) objects.
...
 ```
 
 ### `repository-code`
 
-- type: ...
+- type: [`definitions.url`](#definitionsurl)
 - required: false
 
-...
+The URL of the software in a source code repository.
```
Comment (jspaaks) [inline schema-guide.md:346]: What about

```suggestion
The URL of the work in a source code repository (when the work is software).
```

File: schema-guide.md [hunk 3]
```diff
@@ -313,87 +314,93 @@ Array of [`definitions.reference`](#definitionsreference) objects.
...
 
 ```yaml
+title: "cffconvert"
 ```
 
 ### `type`
 
-- type: ...
+- type: enum (`"software"` or `"dataset"`)
+- default: `"software"`
```
Comment (jspaaks) [inline schema-guide.md:370]:
```suggestion
- type: enum (`software` or `dataset`)
- default: `software`
```

Can we do without the quotes? (Also in the example)

**Comment by jspaaks** [comment]
I left a couple of comments, but good to merge either way.

BTW I just noticed this was still draft, sorry if I interfered with your business prematurely.

File: schema-guide.md (hunk 3 above)
Comment (sdruskat) [reply schema-guide.md:370]: Sure can do, although this obfuscates the string-ity a tad, but that's fine with me (and any confusion is entirely YAML's fault).

**Comment by sdruskat** [comment]
> 
> 
> I left a couple of comments, but good to merge either way.
> 
> BTW I just noticed this was still draft, sorry if I interfered with your business prematurely.

No worries, yea, wasn't done quite yet, but good enough to merge this now with your suggestions (after #242 has been merged), and start a new one (with a better description I guess).

**Comment by sdruskat** [comment]
Closing as superseded by #249

**Pull request closed without merging**